### PR TITLE
Add basic order methods

### PR DESCRIPTION
**Pull request by Jurrie**
- create_order(item_id, item_count)
- get_order_status(order_id)
- abort_order(order_id)

### CODE REVIEW

**Comment by Jurrie** [comment]
This will only allow you to create orders, not to pay them. For this, you'll need additional methods.

**Comment by Leon-Sander** [comment]
I tried the create_order method, and it reserved the bag for me. But it was not visible in the app, I could not pay since in the app is no state of reserved and not paid, only paid which then is reserved. Do you know how it is possible to pay at this point?

**Comment by Jurrie** [comment]
Indeed; the app uses the create_order method to reserve a bag. After that, additional methods are invoked to pay for the order. These methods are different for every payment method. I only use one payment method, and didn't bother yet to dissect all those methods.

I thought I'd put these methods in PR first, and wait to see if these get merged. After that, and when I have time (and feel like it) I'll have a crack at the methods needed for payment of orders. But since my PR is stale for a month now, I doubt it will get merged any time soon.

But, after reserving the bag, did you check "my account" (I mean the bottom right button) → "Mag Bags & Parcels saved"? Maybe it's listed there? (I don't know for sure if it should be listed there, just guessing.)

File: README.md [hunk 1]
```diff
@@ -416,6 +419,85 @@ print(item)
...
+  },
+  "reserved_at": "2023-01-01T10:30:32.331280392",
+  "order_type": "MAGICBAG"
+}
+```
+
+</details>
+
+Please note that payment of an order is currently not implemented.
+In other words: you can create an order via this framework, but you can not pay for it.
```
Comment (ahivert) [inline README.md:469]:
```suggestion
In other words: you can create an order via this client, but you can not pay for it.
```

**Review by ahivert — CHANGES_REQUESTED** [review]
Cool ! Thank you ! Sorry for the delay, just small remarks and this will be merged :)

**Comment by Leon-Sander** [comment]
> Indeed; the app uses the create_order method to reserve a bag. After that, additional methods are invoked to pay for the order. These methods are different for every payment method. I only use one payment method, and didn't bother yet to dissect all those methods.
> 
> I thought I'd put these methods in a PR first, and wait to see if these get merged. After that, and when I have time (and feel like it) I'll have a crack at the methods needed for payment of orders. But since my PR is stale for a month now, I doubt it will get merged any time soon.
> 
> But, after reserving the bag, did you check "my account" (I mean the bottom right button) → "Mag Bags & Parcels saved"? Maybe it's listed there? (I don't know for sure if it should be listed there, just guessing.)

I Just tried it and unfortunately they are not listed in the app. Do you have a payment method implemented for yourself and could share it? It would be amazing to be able to use it. Great work so far.

**Comment by Jurrie** [comment]
> Do you have a payment method implemented for yourself and could share it? It would be amazing to be able to use it. Great work so far.

I do have a payment method implemented. But it's very rough code. I really should clean it up, add some safety checks, add some documentation, make it tidy 😀 I can't give any promises on when that will be done.

In the mean time, you could try to figure out the API calls for yourself. It would be a more long-term solution anyway. Please check [the reply I gave](https://github.com/ahivert/tgtg-python/issues/215#issuecomment-1476785324) on the question of solve-fx. I think intercepting app traffic is a nice skill to have, and not all that hard. Quite fun actually! 😀

**Comment by Vivalemuc** [comment]
@Jurrie 

Can you share the minimalist snipet to pay an order ? Just to see the mechanism !

Thanks :)

**Pull request merged**